controllers/etcd_controller.go

```diff
@@ -274,7 +273,7 @@ func (r *EtcdReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}, nil
 }
 
-func (r *EtcdReconciler) reconcileServices(etcd *druidv1alpha1.Etcd) (*corev1.Service, error) {
+func (r *EtcdReconciler) reconcileServices(etcd *druidv1alpha1.Etcd, values *map[string]interface{}) (*corev1.Service, error) {
```

Comment:
Does values have to be a pointer? `maps` are already pointers.

Reply:
done